shotgun_api3/lib/mockgun/mockgun.py

```diff
@@ -613,7 +622,7 @@ def _compare(self, field_type, lval, operator, rval):
             elif operator == "contains":
                 return rval in lval
             elif operator == "not_contains":
-                return lval not in rval
+                return rval not in lval
```

Comment:
This is a behavior change, it seem the "not_contains" in mockgun just wasnt working right.